arviz/stats/stats.py

```diff
@@ -305,16 +305,32 @@ def _ic_matrix(ics, ic_i):
     return rows, cols, ic_i_val
 
 
-def hpd(ary, credible_interval=None, circular=False, multimodal=False, skipna=False):
+def hpd(
+    ary,
+    *,
+    group="posterior",
+    var_names=None,
+    credible_interval=None,
+    circular=False,
+    multimodal=False,
+    skipna=False,
+    **kwargs
```

Comment:
should be documented too

Reply:
I have documented kwargs. I have specified the return type to be depending on the input.